tensorrt_llm/_torch/pyexecutor/_util.py

```
@@ -180,6 +180,12 @@ def _get_token_num_for_estimation(self) -> int:
             num_cache_blocks += (num_req_tokens +
                                  executor_config.tokens_per_block -
                                  1) // executor_config.tokens_per_block
+
+        cuda_graph_warmup_block = (
```

Ruff (0.12.2)

Local variable `cuda_graph_warmup_block` is assigned to but never used

Remove assignment to unused variable `cuda_graph_warmup_block`

(F841)